velox/exec/TaskStructs.h

```diff
@@ -89,6 +89,8 @@ struct LocalExchangeState {
 struct SplitGroupState {
   /// Map from the plan node id of the join to the corresponding JoinBridge.
   std::unordered_map<core::PlanNodeId, std::shared_ptr<JoinBridge>> bridges;
```

Comment:
This can only hold `HashJoinBridge` now, let's update the type to reflect this

Reply:
This `bridges` map is used for storing `NestedLoopJoinBridge` too. So, the type needs to be `JoinBridge`.

Reply:
Can you add a comment to document that it will contains only `HashJoinBridge` and `NestedLoopJoinBridge`?

Reply:
done.